### ISSUE TITLE
CSS for flashcard background

### ISSUE THREAD

**Issue by videophonegeek** [curtisblackwell/flash_cards#4]
I realize this is an old project but I used it for some practice exercises for my students.  I even added text to speech when the card is flipped, and images to the paragraph sections.

Legacy paper flashcards are generally blank on one side and lined on the flip side. The existing CSS puts a red line on the card in general, and side_one and side_two both get the blue lines. Simple change to put all the background line drawing css controlled by .side_two, keep the general css for .card_one and .card_two, and remove the red line from .card. Thus a more realistic card is imaged.

**Comment by curtisblackwell** [comment]
@videophonegeek Haha, I haven't seen a real one in decades, and the images I found online didn't show both sides.

If you want to submit a PR, I'm happy to accept it. Or even direct people to a fork if you want to take it over.

**Comment by videophonegeek** [comment]
Yes, the flashcard metaphor is outdated but you wouldn't believe how many techniques are based on it.  Formost is Anki and Memrise. Flashcard stock was available in two sizes: 3x5 and 4x6.  I found visually 4x6 which I expanded to 600 x 1000 gave just a little better room and the existing CSS handles it. The animation is still cool and even if the students never used a paper product it is applicable.  I build the cards from a CSV so easy to create and import.

```
  From: Curtis Blackwell <notifications@github.com>
```

 To: curtisblackwell/flash_cards flash_cards@noreply.github.com 
Cc: John Feagans feagans_john@yahoo.com; Mention mention@noreply.github.com
 Sent: Saturday, July 30, 2016 2:19 PM
 Subject: Re: [curtisblackwell/flash_cards] CSS for flashcard background (#4)

@videophonegeek Haha, I haven't seen a real one in decades, and the images I found online didn't show both sides.If you want to submit a PR, I'm happy to accept it. Or even direct people to a fork if you want to take it over.—
